src/style.css

```diff
@@ -1,3 +1,75 @@
+:root {
+  --color-primary: #00acdc;
+  --logo-size: 40px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: #000;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.photo {
+  width: var(--logo-size);
+  height: var(--logo-size);
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+}
+
+.nav__item {
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  padding: 7px 0;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__item a {
+  text-decoration: none;
+  color: #000;
```

Comment:
put this color also to the variable, because it's used more than once in the code